board_client/uw/wifi_service.py

```
@@ -25,11 +25,14 @@ async def connect_wifi():
-    return False
+    raise RuntimeError("WiFi connection failed")
 
 async def wifi_monitor():
     while True:
         if not wlan.isconnected():
             state.wifi_connected = False
-            await connect_wifi()
+            try:
+                await connect_wifi()
+            except RunTimeError:
+                pass
```

Ruff (0.11.9)

Use `contextlib.suppress(RunTimeError)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(RunTimeError)`

(SIM105)